_posts/2024-10-14-Introduction-of-new-opensearch-website-search.markdown

```diff
@@ -0,0 +1,71 @@
+date: 2024-10-14 
+categories:
+  - community-updates
+  - update
+  - community
+meta_keywords: OpenSearch search update, improved search results, advanced filters, documentation search, OpenSearch search page, Amazon OpenSearch Service, version-specific search, search infrastructure update
+meta_description: Discover the new and improved search functionality on OpenSearch.org. Learn how our updated infrastructure, advanced filters, and dedicated search results page make it easier to find the documentation, blog posts, and event information you need.
+---
+We're pleased to announce an update to the search functionality on OpenSearch.org. This new design addresses key challenges in the current infrastructure and search experience while setting the stage for more advanced search features that help you find the information you need more easily.
+
+## The new search functionality
+
+The previous search system had limitations related to scalability, maintenance, and user experience. With this update, we're moving to Amazon OpenSearch Service for a more reliable and scalable infrastructure, and we've added new search features to make your experience faster and more efficient.
+
+**Here's what's new:**
+
+- [**Improved infrastructure**](#better-search-infrastructure) to eliminate deployment and indexing issues.
+- **No more duplicate results**, giving you cleaner and more accurate search results.
+- A [**new search results page**](#new-search-results-page) that goes beyond the dropdown format.
+- **Filters for documentation, blog posts, and events** to help you focus on relevant content.
+- The ability to **search archived versions** of documentation.
+
+## Better search infrastructure
+
+We've restructured the backend of our search system to improve both performance and relevance. Now content is indexed by documentation version, so your search results are version specific. This reduces clutter and duplication, making it easier for you to find the right information quickly.
+
+We've also upgraded from basic query strings to `match` and `multi_match` queries, which provide more precise results by searching across multiple fields. Additionally, we've applied `indices_boost` to prioritize documentation in search results, further improving relevance.
+
+## New search results page
+
+We've moved away from the simple dropdown search interface in favor of a full search results page, offering a more powerful and user-friendly way to navigate search results. This page gives you more control over your search experience and makes it easier to refine your results.
+
+## Key features
+
+Here's a look at the key improvements and new features that enhance your search experience on OpenSearch.org:
+
+- **Dedicated search results page**:
+    After submitting a search, you'll be taken to a new page that shows a more comprehensive list of results than was provided by the previous dropdown. You can browse through these results more easily, without leaving the page.
```

Comment:
Suggested text:
Search results are shown on a dedicated page with a comprehensive list of results, which makes it easy to review them.